eagle-external/eagle-cdh/src/aux/eagle-server.sh

```diff
@@ -0,0 +1,135 @@
+#!/bin/bash
```

Comment:
while packaging this module, can you pick up eagle.conf and copy it as eagle.properties via  maven-assembly-plugin since the file content dont change ?
btw, i dont see eagle.properties  file in this PR. is it in different format than eagle.conf?